v2/product/products.md

```diff
@@ -0,0 +1,16 @@
+# Products
+
+... needs picture of product admin ...
+
+In the new architecture, a Product entity references one or more ProductVariation entities. Thus the product entities
+replace D7 product display nodes (and match the D8 nodes visually) while the variation entities replace D7 product
```

Comment:
Yes, as I said in person, these notes are roughly copied from blog posts. Really appreciated the idea of using the tests to form the documentation.